velox/functions/sparksql/specialforms/SparkCastHooks.h

```diff
@@ -59,5 +62,14 @@ class SparkCastHooks : public exec::CastHooks {
   }
 
   exec::PolicyType getPolicy() const override;
+
+ private:
+  const core::QueryConfig& config_;
+  TimestampToStringOptions timestampToStringOptions_ = {
```

Comment:
nit: document this variable. Perhaps move the documentation for 'timestampToStringOptions()' here and also modify the description on timezone.